### ISSUE TITLE
Image annotation image blocks appear to lack alt text

### ISSUE THREAD

**Issue by BuddyinKC** [omeka-s-modules/ImageAnnotate#6]
We are testing out the Image Annotate module. It is pretty cool! However, WAVE is flagging the output for lacking alt text. The media itself has alt text. However, that does not appear in the <img> element on the page. Example: https://exhibits-umkc.mobiusconsortium.org/s/paintedworlds/page/group1

**Comment by kimisgold** [comment]
Thanks for catching this. We're working on a solution that should be out soon in a bugfix release.

**Comment by jimsafley** [comment]
Fixed in 95c0d9cefde53122e5560f6c1620a2f37bc26f3d
